tree/tree/src/TBranchElement.cxx

```diff
@@ -1053,9 +1053,10 @@ void TBranchElement::Browse(TBrowser* b)
                // we can only find out asking the streamer given our ID
                TStreamerElement *element=nullptr;
                TClass* clsub=nullptr;
-               if (fID>=0 && GetInfoImp()
-                   && GetInfoImp()->IsCompiled()
-                   && ((element=GetInfoImp()->GetElement(fID)))
+               auto infoimp = (fID >= 0) ? GetInfoImp() : nullptr;
```

Comment:
```suggestion
               auto info = (fID >= 0) ? GetInfoImp() : nullptr;
```
(`GetInfoImp[l]` is stand for `GetInfo via the Internal/Private Implementation`)